package.json

```diff
@@ -23,22 +23,26 @@
   },
   "devDependencies": {
     "@babel/cli": "^7.6.3",
-    "@babel/core": "^7.6.3",
+    "@babel/core": "^7.17.5",
+    "@babel/eslint-parser": "^7.17.0",
     "@babel/preset-env": "^7.6.3",
     "@babel/preset-flow": "^7.0.0",
     "@babel/register": "^7.7.4",
     "@commitlint/cli": "^12.1.1",
     "@commitlint/config-conventional": "^12.1.1",
     "@istanbuljs/nyc-config-babel": "^2.1.1",
+    "@stacks/eslint-config": "^1.2.0",
     "babel-eslint": "^10.0.3",
     "babel-plugin-istanbul": "^5.2.0",
     "dotenv": "^8.2.0",
     "eslint": "^6.7.1",
+    "eslint-plugin-prettier": "^4.0.0",
```

Comment:
as this is already declared in the "@stacks/eslint-config" https://github.com/hirosystems/eslint-config/blob/main/package.json#L34 I think you can remove it :)